Audacia.Quest/Audacia.Quest.Core/Params/Keyboard.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Audacia.Quest.Core.Params
+{
+    public class Keyboard
+    {
+        public int KeyCode { get; set; }
```

Comment:
Just a suggestion but, for classes that are used as event data it may look better to make their properties read only, and only set their data in their constructor.

For example usage of this could then look like
```
[JSInvokable]
public async ValueTask OnKeyUp(int keyCode)
{
   EventsCollection.PerformEvent(EventConstants.KEY_UP, new Keyboard(keyCode));
}
```